composer.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
interesting why file was not present in project by default.

Reply:
put all project files in git ?